pages/index.js

```diff
@@ -61,3 +30,37 @@ const Button = styled.button`
   margin: 2px;
   padding: 0.5px;
 `;
+
+export default function Landingpage() {
+  const router = useRouter();
+  const [output, setOutput] = useState("");
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const entry = event.target.elements.text.value;
+    setOutput(entry);
+  };
+  return (
+    <>
+      <Title>Your Travel Journal</Title>
+      <Form onSubmit={handleSubmit}>
+        <fieldset>
+          <legend>Where is your journey going?</legend>
+          <Label htmlFor="text">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="text"
+            name="text"
+            placeholder="enter your destination..."
+          />
+          <Button type="submit">Submit</Button>
+          <p>{output}</p>
+        </fieldset>
+      </Form>
+
+      <button onClick={() => router.push("/EntryListPage")}>
+        <p>Entrylist</p>
+      </button>
```

Comment:
This is a link and should be a Link (or a-Tag) component instead of a button for accessibility and semantics.